src/main/scala/ai/privado/utility/ImportUtility.scala

```diff
@@ -0,0 +1,106 @@
+package ai.privado.utility
+
+import scala.util.control.Breaks._
+import java.nio.charset.MalformedInputException
+import scala.io.Source
+import ai.privado.model.Language
+
+import scala.collection.parallel.CollectionConverters.ImmutableIterableIsParallelizable
+
+object ImportUtility {
+
+  private var allImports: Set[String] = Set();
+
+  // get language specific import matching regex
+  private def getMatchImportRegex(language: Language.Value): String = {
+    val result = language match {
+      case Language.JAVA       => "^\\s*(import)\\s+.*$"
+      case Language.PYTHON     => "^\\s*(import)\\s+.*$"
+      case Language.JAVASCRIPT => "^\\s*(import|require)\\s+.*$"
+      case default             => "(import)\\s+" // Default is java
+    }
+    return result;
+  }
+
+  private def getFileExtension(language: Language.Value): String = {
+    val result = language match {
+      case Language.JAVA       => ".java"
+      case Language.PYTHON     => ".py"
+      case Language.JAVASCRIPT => ".js"
+      case default             => ".java" // Default is java
+    }
+
+    return result;
+  }
+
+  // get regex to match only import words
+  private def getLiteralWordRegex(language: Language.Value): String = {
```

Comment:
Instead of having 2 different function to return value based on Language, you can create a class and use that to get these 2 values `getMatchImportRegex, getLiteralWordRegex`, along with any other lang specific values like multi line comment dilimeter